packages/flutter/lib/src/rendering/box.dart

```diff
@@ -1875,21 +1886,21 @@ abstract class RenderBox extends RenderObject {
   ///
   /// When implementing a [RenderBox] subclass, to override the baseline
   /// computation, override [computeDistanceToActualBaseline].
-  double getDistanceToBaseline(TextBaseline/*!*/ baseline, { bool onlyReal = false }) {
+  double? getDistanceToBaseline(TextBaseline baseline, { bool onlyReal = false }) {
     assert(!_debugDoingBaseline, 'Please see the documentation for computeDistanceToActualBaseline for the required calling conventions of this method.');
-    assert(!debugNeedsLayout);
+    assert(!debugNeedsLayout!);
     assert(() {
       final RenderObject parent = this.parent as RenderObject;
```

Comment:
how did `parent` end up non-nullable?

Reply:
Changed to nullable

Reply:
I'm just amazed that tests passed and such with it being non-nullable :-/